app/src/main/java/com/babylon/wallet/android/presentation/transfer/TransferViewModel.kt

```diff
@@ -486,6 +497,10 @@ class TransferViewModel @Inject constructor(
                 get() = maxAccountAmount < 1.toDecimal192()
         }
     }
+
+    sealed interface Event : OneOffEvent {
+        data class OnAssetClicked(val asset: SpendingAsset, val fromAccount: Account) : Event
```

Comment:
`OnAssetClicked` may be renamed to ShowAssetDetails or something similar, because this name should reflect the operation the UI has to perform as a result of an asset being clicked.

Reply:
You are right. Will change